extensions/qute/component/src/generated/resources/org/apache/camel/component/qute/qute.json

```diff
@@ -1,38 +1,39 @@
 {
-  "component": {
```

Comment:
Is there any need to have the `qute.json` file modified in this pr ?

Reply:
No, I think we don't need this change.